src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
                 }
             ],
             "S": [
+                {
+                    "term": "SARS-CoV-2",
+                    "anchor": "sars_cov_2",
+                    "description": "See <a href='#glossary_corona_virus'>Corona Virus</a>"
+                },
+                {
+                    "term": "Science-Blog",
+                    "anchor": "science_blog",
+                    "description": "The Corona-Warn-App Science Blog covers various scientific topics related to the Corona-Warn-App, such as its effectiveness, benefits, and users, as well as results of scientific evaluations conducted by the Robert Koch Institute."
+                },
+                {
+                    "term": "Short Lived Random Bluetooth ID",
+                    "anchor": "short_lived_random_bluetooth_id",
+                    "description": "Unique key generated several times an hour, derived from the random device key and exchanged between neighboring smartphones via Bluetooth."
+                },
                 {
                     "term": "Signature",
                     "anchor": "signature",
-                    "description": "A digital signature is a signature that is created by a cryptographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> <br/><br/> As a consequence, the signature must fulfill the following requirements: <br/><ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
+                    "description": "A digital signature is a signature that is created by a crytographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> As a consequence, the signature must fulfill the following requirements: <ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"

```

Comment:
The typo correction "cryptographic" is overwritten by this change. Please revert to the correct spelling. "crytographic" is wrong.

Reply:
Will be corrected in the next commit, thanks!